scss/_off.scss

```diff
@@ -325,3 +325,5 @@ p.details {
   color: gray;
   font-size: 0.8em;
 }
+
+@media only screen and (max-width: 40em) { h2 { font-weight:bold; font-size:1.2em;} }
```

Comment:
Did you test if this messes up `h2` tags on other OFF pages?

Reply:
@stephanegigandet

Reply:
@hangy short answer, don't think was tested

Reply:
Well if it should actually fix the h2 tags in other pages. On small screens, they currently do not differ from normal text at all.